controller/iteration_blackbox_test.go

```diff
@@ -612,21 +611,22 @@ func (rest *TestIterationREST) createIterations() (*app.IterationSingle, *accoun
 	return created, owner
 }
 
-// TestIterationActivedByUser tests iteration should always be active when user sets it to active
+// TestIterationActivedByUser tests:
+// 1. Iteration should be active when it is in timeframe
+// 2. Iteration should not be active when it is outside the timeframe
+// 3. Iteration should always be active when user sets it to active
 func (rest *TestIterationREST) TestIterationActivatedByUser() {
 	itr1, owner := rest.createIterations()
 	assert.Equal(rest.T(), false, *itr1.Data.Attributes.UserActive)
```

Comment:
Do you want to use `iteration.IterationNotActive` instead of `false`? Just thinking.

Reply:
fixed it here e4ffadb